test/Microsoft.AspNet.Mvc.Test/MvcOptionsSetupTest.cs

```diff
@@ -134,7 +137,7 @@ public void Setup_IgnoresAcceptHeaderHavingWildCardMediaAndSubMediaTypes()
         }
 
         [Fact]
-        public void Setup_SetsUpExcludeFromValidationDelegates()
+        public void Setup_SetsUpMetadataDetailsProviders()
```

Comment:
I'm really not a fan of this kind of test in general. I updated it, but IMO these are super low value.